### PR TITLE
Bump to 2.1.1

### PR DESCRIPTION
**Pull request by ocefpaf**
Testing latest version.

PS: Maybe we should do `1.11.5` too...

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@gillins Any idea why in the windows python 2.7 builds the xz-archives cannot be extracted?

**Comment by gillins** [comment]
Yikes. I've got a terrible feeling this is related to https://github.com/conda-forge/libxml2-feedstock/pull/15. I thought there was a reason that we weren't to upgrade, but I couldn't remember the details - maybe something to do with that version of `xz` not building with VS2008? @ocefpaf will know I hope...

File: recipe/build.sh [hunk 1]
```diff
@@ -77,7 +77,8 @@ export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
             --with-python \
             $OPTS
 
-make >> $BUILD_OUTPUT 2>&1
+# CircleCI offers two cores.
+make -j 2 >> $BUILD_OUTPUT 2>&1
```
Comment (kmuehlbauer) [inline recipe/build.sh]: Neat, this did the trick on Circle wrt to maximum build time (120m).

**Comment by gillins** [comment]
Seems that `xz` 5.2 is VS 2015 (Python 3.5) only: https://github.com/bminor/xz/blob/master/windows/INSTALL-MSVC.txt, https://github.com/conda-forge/xz-feedstock/blob/master/recipe/meta.yaml

**Comment by ocefpaf** [comment]
> Seems that `xz` 5.2 is VS 2015 (Python 3.5) only: https://github.com/bminor/xz/blob/master/windows/INSTALL-MSVC.txt, https://github.com/conda-forge/xz-feedstock/blob/master/recipe/meta.yaml

Ouch! And I thought I was being smart by using a smaller tarball.

**Comment by gillins** [comment]
Ah right - it seems `libxml2` only depends on `xz` on Unix, not Windows so you are right - not a problem.

**Comment by ocefpaf** [comment]
Wow! `AppVeyor` is finishing before `CircleCI` after the upgrade!

**Comment by kmuehlbauer** [comment]
> Wow! AppVeyor is finishing before CircleCI after the upgrade!

You are referring to AppVeyor's recent move to RackSpace? They did a great job, this is a real improvement.

**Comment by ocefpaf** [comment]
> You are referring to AppVeyor's recent move to RackSpace? They did a great job, this is a real improvement.

Yep. It is still a single, but the infrastructure is much better.

**Comment by ocefpaf** [comment]
We have #91 and #83 opened, so I won't merge this before I can check if I can fix those here to avoid multiple builds in the channel.

**Comment by kmuehlbauer** [comment]
> We have #91 and #83 opened, so I won't merge this before I can check if I can fix those here to avoid multiple builds in the channel.

Sure, #91 might be already fixed due to new kealib version, and you also did a fiona-update recently, @ocefpaf?

Don't know how to tackle #83, though.

**Comment by ocefpaf** [comment]
> Sure, #91 might be already fixed due to new kealib version

I am in transit today, but as soon as I settle in the airport I will confirm that.

> and you also did a fiona-update recently, @ocefpaf?

Yep. Should be working. If not let me know :grimacing: 

> Don't know how to tackle #83, though.

I want to at least add a test to understand what is going on.

**Comment by pelson** [comment]
LGTM 👍

**Comment by ocefpaf** [comment]
I was waiting on more comments in #95, but I guess there is no point. We either have to split like `defaults` or keep it in as 1 package. I will send another PR for the split later next week.

**Pull request merged**